osu.Game.Tournament/IO/TournamentStorage.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Logging;
+using osu.Framework.Platform;
+using osu.Game.IO;
+using System.IO;
+using osu.Game.Tournament.Configuration;
+
+namespace osu.Game.Tournament.IO
+{
+    public class TournamentStorage : MigratableStorage
+    {
+        private readonly Storage storage;
+        internal readonly TournamentVideoResourceStore VideoStore;
```

Comment:
The `internal` looks really strange to me. I'd write this as

```suggestion
        public readonly TournamentVideoResourceStore VideoStore { get; }
```